src/command.rs

```diff
@@ -91,15 +92,19 @@ impl Command {
         command.arg("--");
         command.arg(url);
 
-        // NOTE: On Windows, browser spawns process into a Job object.
-        // NOTE: We need to detach player from the job, so it won't get killed after we're done,
-        // NOTE: See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging#closing_the_native_app
+        // NOTE: Make sure the subprocess is not killed.
+        //       See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Native_messaging#closing_the_native_app
+    
+        #[cfg(target_family = "unix")]
```

Comment:
I think it would be better to extract that into a function.
```rust
impl Command {
    // ...

    #[cfg(target_family = "unix")]
    fn detach_player(command: &mut Command) {}
  
    #[cfg(target_family = "windows")]
    fn detach_player(command: &mut Command) {}
 }
```

Reply:
By `command: &mut Command` I assume you mean `std::process::Command` and not `ff2mpv_rust::command::Command`. The function call would look like:

```rust
impl Command {
    fn launch_mpv(command: String, args: Vec<String>, url: &str) -> Result<(), io::Error> {
        let mut command = todo!();
        Command::detach_player(&mut command);
        todo!(command);
    }
}
```

Is that what you mean?

Reply:
IMO, the current approach is cleaner. It also removes the need for `#[cfg(not(any(unix, windows)))]`